v2/spiffetls/tlsconfig/config.go

```diff
@@ -89,25 +159,63 @@ func MTLSWebServerConfig(cert *tls.Certificate, bundle x509bundle.Source, author
 // it will be wrapped by by this package and invoked after SPIFFE
 // authentication has completed.
 func HookMTLSWebServerConfig(config *tls.Config, cert *tls.Certificate, bundle x509bundle.Source, authorizer Authorizer) {
-	panic("not implemented")
+	config.InsecureSkipVerify = true
+	config.GetClientCertificate = func(*tls.CertificateRequestInfo) (*tls.Certificate, error) {
+		return cert, nil
+	}
+	config.VerifyPeerCertificate = wrapVerifyPeerCertificate(config.VerifyPeerCertificate, VerifyPeerCertificate(bundle, authorizer))
+
+	// No used by server
+	config.Certificates = nil
+	config.RootCAs = nil
+	config.NameToCertificate = nil
 }
 
 // GetCertificate returns a GetCertificate callback for tls.Config. It uses the
 // given X509-SVID getter to obtain a server X509-SVID for the TLS handshake.
 func GetCertificate(svid x509svid.Source) func(*tls.ClientHelloInfo) (*tls.Certificate, error) {
-	panic("not implemented")
+	return func(*tls.ClientHelloInfo) (*tls.Certificate, error) {
+		return getTLSCertificate(svid)
+	}
 }
 
 // GetClientCertificate returns a GetClientCertificate callback for tls.Config.
 // It uses the given X509-SVID getter to obtain a client X509-SVID for the TLS
 // handshake.
 func GetClientCertificate(svid x509svid.Source) func(*tls.CertificateRequestInfo) (*tls.Certificate, error) {
-	panic("not implemented")
+	return func(*tls.CertificateRequestInfo) (*tls.Certificate, error) {
+		return getTLSCertificate(svid)
+	}
 }
 
 // VerifyPeerCertificate returns a VerifyPeerCertificate callback for
 // tls.Config. It uses the given bundle source and authorizer to verify and
 // authorize X509-SVIDs provided by peers during the TLS handshake.
 func VerifyPeerCertificate(bundle x509bundle.Source, authorizer Authorizer) func([][]byte, [][]*x509.Certificate) error {
-	panic("not implemented")
+	return func(raw [][]byte, _ [][]*x509.Certificate) error {
```

Comment:
When wrapping, we should probably not use this callback implementation since we'll want to take the `certs` returned from `x509svid.ParseAndVerify` and provide them as the `verifiedChains` to the wrapped callback.